blocks/identity-block/features/one-time-password/default.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { useFusionContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+import { Input, useIdentity, BotChallengeProtection, Paragraph, Heading, HeadingSection } from "@wpmedia/arc-themes-components";
+import HeadlinedSubmitForm from "../../components/headlined-submit-form";
+
+const BLOCK_CLASS_NAME = "b-one-time-login-form";
+
+const errorCodes = {
+	100015: "identity-block.login-form-error.account-is-disabled",
+	130001: "identity-block.login-form-error.captcha-token-invalid",
+	130051: "identity-block.login-form-error.unverified-email-address",
+	100013: "identity-block.login-form-error.max-devices",
+	0: "identity-block.login-form-error.invalid-email-password",
+};
+
+const definedMessageByCode = (code) => errorCodes[code] || errorCodes["0"];
+
+const OneTimePasswordLogin = () => {
+	const { arcSite } = useFusionContext();
+	const { locale } = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(locale);
+	const { Identity, isInitialized } = useIdentity();
+	const [captchaToken, setCaptchaToken] = useState();
+	const [resetRecaptcha, setResetRecaptcha] = useState(true);
+	const [error, setError] = useState();
+	const [captchaError, setCaptchaError] = useState();
+	const [success, setSuccess] = useState(false);
+	const [userEmail, setUserEmail] = useState('');
+
+	if (!isInitialized) {
+		return null;
+	}
+
+	if (success) {
+		return (
+			<div className={BLOCK_CLASS_NAME}>
+				<HeadingSection>
+					<Heading>{phrases.t("identity-block.ota-success-heading")}</Heading>
```

Comment:
The font and font size are different from figma on both `Check your email title` and `Request one time log in link`

Reply:
Okay, I'll check.